packages/ckeditor5-utils/src/translation-service.ts

```diff
@@ -114,18 +111,33 @@ if ( !global.window.CKEDITOR_TRANSLATIONS ) {
 export function add(
 	language: string,
 	translations: { readonly [ messageId: string ]: string | ReadonlyArray<string> },
-	getPluralForm?: ( n: number ) => number
+	getPluralForm?: ( n: number ) => number,
+	editor?: Editor
```

Comment:
What is the purpose?
`add` function was supposed to amend the global variable with new translations. `editor` will have the config already initialized with all the provided translations. How is `add` useful for configuring the specific instance of editor?